test/test_helper.rb

```diff
@@ -1,3 +1,8 @@
 require 'minitest'
 require 'simplecov'
 SimpleCov.start
+require 'hurley'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.